mob.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "0.0.24",
+  "homepage": "https://github.com/remotemobprogramming/mob",
+  "checkver": "github",
+  "url": "https://github.com/remotemobprogramming/mob/releases/latest",
```

Comment:
url should probably be https://github.com/remotemobprogramming/mob/releases/tag/v0.0.24

Reply:
That was the first URL I was using, but it should be able to handle "hey, there's a new version, do you want to update" type of thing pointing at latest. But we can certainly change it to start with just pointing at 24.